adapters/adnuntius/adnuntius.go

```diff
@@ -410,6 +447,13 @@ func generateAdResponse(ad Ad, imp openrtb2.Imp, html string, request *openrtb2.
 		}
 	}
 
+	extJson, err := generateReturnExt(ad, request)
+	if err != nil {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("Error extracting Ext: %s", err.Error()),
+		}}
+	}
```

Comment:
please add test for this. Not covered 
![image](https://github.com/user-attachments/assets/77d15955-9e06-4166-9528-5fa71e55c7a2)